src/GafferUI/RotateHandle.cpp

```diff
@@ -152,18 +152,13 @@ Imath::Eulerf RotateHandle::rotation( const DragDropEvent &event )
 
 	float rotate = ( closestRotation( m_drag.position( event ), m_rotation ) - closestRotation( m_drag.startPosition(), 0.0f ) );
 
-	// snap to 30 degree increments
+	// snap
 	if( event.modifiers & ButtonEvent::Control )
 	{
-		float piOverSix = M_PI / 6.0;
-		rotate /= piOverSix;
-		rotate = std::round( rotate ) * piOverSix;
-	}
-
-	// precision mode
-	if( event.modifiers & ButtonEvent::Shift )
-	{
-		rotate *= 0.1;
```

Comment:
This old scaling worked in angular space, and the new one works in planar space. I much prefer the old angular one, as it meant that precise mode felt the same as regular mode, and you could keep "dialling around" in precise mode.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3438/commits/ef6fcaa3fa26c42365f4d499f357550644c98cde via https://github.com/GafferHQ/gaffer/pull/3438/commits/732c792dcce62c32b6776b56ce685274a6042738.